src/renderer/fonts/font_loader.rs

```diff
@@ -17,7 +21,13 @@ impl FontPair {
     fn new(mut skia_font: Font) -> Option<FontPair> {
         skia_font.set_subpixel(true);
         skia_font.set_hinting(FontHinting::Full);
-        skia_font.set_edging(Edging::AntiAlias);
+
+        let settings = SETTINGS.get::<WindowSettings>();
+        if settings.font_subpixel_antialiasing {
+            skia_font.set_edging(Edging::SubpixelAntiAlias);
+        } else {
+            skia_font.set_edging(Edging::AntiAlias);
```

Comment:
Maybe option should expose all three variants:

Alias, AntiAlias and Subpixel?

And should be named "font_edging" instead of current name.